ouroboros-network-testing/src/Ouroboros/Network/Testing/Data/Script.hs

```diff
@@ -218,9 +218,9 @@ interpretPickScript :: (MonadSTM m, Ord peeraddr)
                     -> STM m (Set peeraddr)
 interpretPickScript scriptVar available pickNum
   | Set.null available
-  = error "interpretPickScript: given empty map to pick from"
+  = error "interpretPickScript: given empty map to pick from" -- TODO
   | pickNum <= 0
-  = error "interpretPickScript: given invalid pickNum"
+  = error "interpretPickScript: given invalid pickNum" -- TODO
```

Comment:
`interpretPickScript` is used primarily to initialize `PickPolicy` fields of a `PeerSelectionPolicy` (in `ouroboros-network:Test.Ouroboros.Network.PeerSelection.MockEnvironment`). If `pickPeers` can reliably be expected to serve as the applier of `PickPolicy`s, then I wonder if these checks are duplicative of those in `pickPeers`.

Reply:
Can do.